codespell_lib/data/dictionary.txt

```diff
@@ -10931,6 +10942,7 @@ geoemtry->geometry
 geogcountry->geocountry
 geographich->geographic
 geographicial->geographical
+geoio->geoip
```

Comment:
Rather specific, but probably generic enough to be of use.

Reply:
Done